src/componentprocessor/index.ts

```diff
@@ -149,12 +150,14 @@ async function processXmlTree(
                     nodeDef.children = [...nodeDef.children];
                     baseNode = baseNodeDef.xmlNode!.attr.extends;
                 } else {
-                    // The reference implementation doesn't allow extensions of unknown node subtypes, but
-                    // BRS hasn't implemented every node type in the reference implementation!  For now,
-                    // let's warn when we detect unknown subtypes.
-                    console.error(
-                        `XML component '${nodeDef.name}' extends unknown component '${baseNode}'. Ignoring extension.`
-                    );
+                    if (!(baseNode in BrsComponentName)) {
```

Comment:
This silences this `console.error` for those components that _have_ now been implemented.